test/integrations/destinations/bqstream/router/data.ts

```diff
@@ -225,6 +391,76 @@ export const data = [
                 Name: 'bqstream test',
               },
             },
+            {
+              batched: false,
+              destination: {
+                Config: {
+                  datasetId: 'gc_dataset',
+                  eventDelivery: true,
+                  eventDeliveryTS: 1636965406397,
+                  insertId: 'productId',
+                  projectId: 'gc-project-id',
+                  rudderAccountId: '1z8LpaSAuFR9TPWL6fECZfjmRa-',
+                  tableId: 'gc_table',
+                },
+                Enabled: true,
+                ID: '1WXjIHpu7ETXgjfiGPW3kCUgZFR',
+                Name: 'bqstream test',
+              },
+              error: 'Message Type not supported: identify',
+              metadata: [
+                {
+                  jobId: 3,
+                  userId: 'user12345',
+                },
+              ],
+              statTags: {
+                destType: 'BQSTREAM',
+                errorCategory: 'dataValidation',
+                errorType: 'instrumentation',
+                feature: 'router',
+                implementation: 'native',
+                module: 'destination',
+              },
+              statusCode: 400,
+            },
+            {
+              batched: true,
```

Comment:
In this test case, only one event is batched. Can we have a some better test cases? 

For ex, Have multiple test events of different users as input and output should be batched events satisfying user level event order constraints.

Reply:
added